server/catgenome/src/main/java/com/epam/catgenome/util/feature/reader/CacheIndex.java

```diff
@@ -0,0 +1,4 @@
+package com.epam.catgenome.util.feature.reader;
+
+public interface CacheIndex {
```

Comment:
It needs a Javadoc, descibing it's purpose, and a better name :)
IndexCache or IndexCacheEntry would be better.